pkg/operator/vttablet/pvc.go

```diff
@@ -24,7 +24,13 @@ import (
 )
 
 // NewPVC creates a new vttablet PVC from a Spec.
+// TODO: "Handle the case when labels are removed from ExtraLabels"
 func NewPVC(key client.ObjectKey, spec *Spec) *corev1.PersistentVolumeClaim {
+	// Store labels in labels obj because we need to add extra label and avoid mutating spec.Labels value
+	labels := map[string]string{}
```

Comment:
We should change `Labels: spec.Labels` below to use this var instead.

Reply:
sorry I do not why it was like this. fixed to use this var